configs/components/fesom/fesom-2.1.yaml

```diff
@@ -63,7 +63,9 @@ pool_dir: "${computer.pool_directories.pool}"
 
 choose_computer.name:
     ollie:
-        pool_dir: "${computer.pool_directories.pool}/../clidyn/"
+        pool_dir: "${computer.pool_directories.pool}/FESOM/"
+        climate_data_dir: "${pool_dir}/hydrography/"
+        forcing_data_dir: "${computer.pool_directories.projects}/clidyn/forcing/"
```

Comment:
Additionally, the new runscript will also work in mistral (and in the future in Levante) because fesom-2.1 already handles correctly the pools there too.